app/renderer/src/actions/Inspector.js

```diff
@@ -139,6 +142,33 @@ const findElement = _.debounce(async function (strategyMap, dispatch, getState,
   return dispatch({type: SET_INTERACTIONS_NOT_AVAILABLE});
 }, 1000);
 
+const isValidGesture = (action) => {
+  if (!action.ticks) {
+    return true;
+  }
+
+  for (const tick of action.ticks) {
+    if (!tick.type || Object.values(POINTER_TYPES).indexOf(tick.type) <= 0) {
+      return false;
+    } else if (
+      tick.type === POINTER_TYPES.POINTER_MOVE &&
+      typeof tick.button == 'undefined' &&
+      !tick.x &&
+      !tick.y
+    ) {
+      return false;
+    } else if (
+      [POINTER_TYPES.POINTER_DOWN, POINTER_TYPES.POINTER_UP].indexOf(tick.type) >= 0 &&
+      typeof tick.button == 'undefined'
+    ) {
+      return false;
+    } else if (tick.type === POINTER_TYPES.PAUSE && !tick.duration) {
```

Comment:
you may simplify and shorten the condition by combining branches using `||` instead of `else if`

Reply:
To promote readability i went ahead with `if/else`. But with the new changes this is not requirement anyways.